docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -109,9 +109,45 @@ method on the key object. Documentation for these methods in found in the
 :mod:`~cryptography.hazmat.primitives.asymmetric.dsa`, and
 :mod:`~cryptography.hazmat.primitives.asymmetric.ec` module documentation.
 
+PKCS12
+~~~~~~
+
+.. currentmodule:: cryptography.hazmat.primitives.serialization.pkcs12
+
+PKCS12 is a binary format described in :rfc:`7292`. It can contain
+certificates, keys, and more. PKCS12 files commonly have a ``pfx`` or ``p12``
+file suffix.
+
+.. note::
+
+    ``cryptography`` only supports a single certificate and private key when
```

Comment:
Well, we support multiple certs, they just have to be in a chain. Maybe phrase this as "only supports a single private key and associated certificates"?